HLTrigger/Configuration/python/HLT_75e33/modules/hltGeneralTracks_cfi.py

```diff
@@ -39,6 +53,19 @@
 from Configuration.ProcessModifiers.trackingLST_cff import trackingLST
 trackingLST.toReplaceWith(hltGeneralTracks, _hltGeneralTracksLST)
 
+_hltGeneralTracksLSTSingleIterPatatrack = hltGeneralTracks.clone(
+    TrackProducers = cms.VInputTag("hltInitialStepTrackSelectionHighPuritypTTCLST", "hltInitialStepTracksT5TCLST", "hltHighPtTripletStepTrackSelectionHighPurity"),
```

Comment:
looking closer : what is in `hltHighPtTripletStepTrackSelectionHighPurity` ? this looks like a legacy recovery unmodified.
I interpreted the plan for the "singleIter" to be literally a single (general) tracking iteration.

Reply:
The LST single iteration is actually "a single iteration of seeds going into LST as input", hence the "Patatrack-only-seeded LST" configuration. An actual single iteration with LST means no building in the endcaps, and I don't think this makes sense at this point (unless Patatrack pixel tracks magically become amazingly better).

Reply:
> An actual single iteration with LST means no building in the endcaps, and I don't think this makes sense at this point

OK, maybe an actual single iteration needs to be added.

What we talked about yesterday or in the past as an ultimate goal is to have a single iteration pata+LST+finalfit and eventually get this to work. Even if it works poorly now, the goal is to make it work and start with having this configuration available.

Reply:
Such a configuration has never been tested but I can test it and show it on Friday. After we see what we get, I can close this PR and open another one with the commit about the single iteration setup with CKF and with an actual single iteration LST. Then we merge that on top of the other PR and we are done (for now). Does this sound like a plan?

Reply:
OK.
sounds like a plan.